contracts/src/interfaces/INativeTokenBridge.sol

```diff
@@ -13,11 +13,16 @@ import {ITeleporterTokenBridge, SendTokensInput} from "./ITeleporterTokenBridge.
  */
 
 /**
- * @dev Interface for a Teleporter token bridge that sends native tokens to another chain.
+ * @notice Interface for a Teleporter token bridge that sends native tokens to another chain.
  */
 interface INativeTokenBridge is ITeleporterTokenBridge {
     /**
-     * @dev Sends native tokens transferred to this contract to the destination token bridge instance.
+     * @notice Receives native tokens transferred to this contract.
+     */
+    receive() external payable;
```

Comment:
My understanding `receive()` is that it's called by default whenever ether is transfer sent to a contract with no calldata. 

Is explicitly including `receive()` in an interface a standard pattern? I'm curious if all contracts by default meet this interface requirement since they must have some default implementation of it even if not explicitly defined. If so, I'm not sure it makes to sense to include in this interface, since it wouldn't have the desired effect of enforcing that any implementations of `INativeTokenBridge` explicitly define it.

Reply:
If a contract does not implement `receive()`, when a call is made to transfer native tokens to the contract with no calldata, the call will fail and revert. Contracts do not meet this interface requirement by default and need to explicity define the function.

Reply:
I would second removing this from the interface. The interface should really be a compile time check, and if a contract inherits this interface as is, and omits `receive()`, then it will compile just fine.

Reply:
sounds good removed since not mandatory for the implementation to have `receive()` implemented. Will leave to the underlying contract.